packages/react-ui-ag/src/Filters/BedroomFilterCard.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+import { RadioGroup } from '@rentpath/react-ui-core'
+import isEqual from 'lodash/isEqual'
+import autobind from 'autobind-decorator'
+import FilterCard from './FilterCard'
+
+@themed(/^BedroomFilter/)
+
+export default class BedroomFilterCard extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    title: PropTypes.string,
+    description: PropTypes.string,
+    applyButtonProps: PropTypes.shape({
+      handleClick: PropTypes.func,
+    }),
+    cancelButtonProps: PropTypes.shape({
```

Comment:
Instead of this you _could_ export the ApplyButton and Cancel button and allow the user to pass the component back in with the extra props attached.  That way they still get the handleClick functionality and separations of concerns is better IMO.